pyramid_oereb/core/readers/real_estate.py

```diff
@@ -73,7 +74,8 @@ def read(self, params, nb_ident=None, number=None, egrid=None, geometry=None):
             plan_for_land_register_main_page_config.get('layer_opacity'),
             Config.get('default_language'),
             Config.get('srid'),
-            Config.get('proxies')
+            Config.get('proxies'),
+            verify_certificate=plan_for_land_register_main_page_config.get('verify_certificate')
```

Comment:
I think "verify_certificate=plan_for_land_register_main_page_config.get('verify_certificate', True)" would be clearer.

Reply:
OK, changed